specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,530 @@
+                "operationId": "FormRecognizer_CustomTrain",
+                "consumes": [
+                    "application/json-patch+json",
+                    "application/json",
+                    "text/json",
+                    "application/*+json"
+                ],
+                "produces": [
+                    "application/json",
+                    "text/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "trainRequest",
+                        "in": "body",
+                        "description": "Request object for training.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/TrainRequest"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "$ref": "#/definitions/TrainResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                  "Train new model example": {
+                    "$ref": "./examples/Train.json"
+                  }
+                },
+            }
+        },
+        "/model/{id}/keys": {
+            "get": {
+                "summary": "Get Keys",
+                "description": "<para>Use the API to retrieve the keys that were\r\n            extracted by the specified model.</para>",
```

Comment:
This `description` will also be used in auto generated source code and API reference.
Please consider plain text or Markdown.

Reply:
Removed <para> tag